.github/workflows/rust.yml

```diff
@@ -53,23 +53,3 @@ jobs:
 
     - name: Clippy (all features enabled)
       run: cargo clippy --all-features -- -D warnings
-
-  Test:
```

Comment:
Is there a reason we can't run cargo test in CI here?

Reply:
I think my reasoning was that we don't really have any useful tests here. But I do think that we should add a testsuite here in the mid-term. Not sure what this would look like? Maybe use russh to create an ssh-agent client? 

Undid the changes though.